projects/linuxkit/linuxkit/patches/0003-pass-cpu-host-when-running-qemu-to-support-testing-a.patch

```diff
@@ -0,0 +1,26 @@
+From caf17c8da1cf7bfeccaa216b1c4d654f2fabb682 Mon Sep 17 00:00:00 2001
+From: Prow Bot <prow@amazonaws.com>
+Date: Mon, 7 Oct 2024 19:35:59 +0000
+Subject: [PATCH 3/3] pass cpu host when running qemu to support testing al23
+
+---
+ src/cmd/linuxkit/run_qemu.go | 3 +++
+ 1 file changed, 3 insertions(+)
+
+diff --git a/src/cmd/linuxkit/run_qemu.go b/src/cmd/linuxkit/run_qemu.go
+index 7a7066264..bd80762f6 100644
+--- a/src/cmd/linuxkit/run_qemu.go
++++ b/src/cmd/linuxkit/run_qemu.go
+@@ -464,6 +464,9 @@ func buildQemuCmdline(config QemuConfig) (QemuConfig, []string) {
+ 		} else {
+ 			qemuArgs = append(qemuArgs, "-cpu", "cortex-a57")
+ 		}
++	} else {
++		// aws - We require x86_64 to run al23 with its version of glibc
++		qemuArgs = append(qemuArgs, "-cpu", "host")
```

Comment:
When testing with a al23 based hook image the default qemu cpu is set to x86_64_v1, but the version of glibc requires v2.  Passing `-cpu host` matches the host version.  This change will not affect our builds only when we run qemu to test it, which is not something our customers do.